docs/source/Tools/Tools.rst

```diff
@@ -867,10 +867,11 @@ The old settings are still active in memory and if something will be saved, only
 This would corrupt the settings file.
 
 
-With only ``USE_SETTINGS_ARCHIVE`` defined during build, the URL and credentials cannot be stored.
-For this the build must be made with ``USE_CUSTOM_PROVISIONING`` defined.
+With only ``FEATURE_SETTINGS_ARCHIVE`` defined during build, the URL and credentials cannot be stored.
+(2022/07/24: Renamed USE_SETTINGS_ARCHIVE to FEATURE_SETTINGS_ARCHIVE)
+For this the build must be made with ``FEATURE_CUSTOM_PROVISIONING`` defined.
 
-N.B. ``USE_CUSTOM_PROVISIONING`` is added on 2022/05/13.
+N.B. ``FEATURE_CUSTOM_PROVISIONING`` is added on 2022/05/13. (2022/07/24: Renamed from USE_CUSTOM_PROVISIONING to FEATURE_CUSTOM_PROVISIONING)

```

Comment:
This will break Custom.h files that are in use.

I'll be probably the only one to actually use it for a project, but still ;)

Reply:
OK, I then expect the same issue with the other renames, I'll revert the renames for backward compatibility, but keep the 0/1 state, as IMHO that's still an improvement.

Reply:
No don't revert it, we have to rename this PR to indicate something like "update Custom.h"
And for sure indicate it in bold on the next release.

Reply:
So what should we (which are using the Custom.h ) do then?
I am always (before the custom Vagrant compilation) checking the difference between current and previous Custom-sample.h and updating my Custom.h file accordingly. Is there any additional task necessary after this PR merge?

Reply:
> Is there any additional task necessary after this PR merge?

That should be it 👍 

Unless of course you have an unpublished local piece of code that is relying on these compile-time flags 😉